plugins/inputs/procstat/README.md

```diff
@@ -53,6 +53,13 @@ Threads related measurement names:
 File descriptor related measurement names (*telegraf* needs to run as **root**):
 - procstat_[prefix_]num_fds value=4
 
+Priority related measurement names:
+- procstat_[prefix_]realtime_priority value=0
+- procstat_[prefix_]nice_priority value=20
+
+Signals related measurement names:
+- procstat_[prefix_]signals_pending value=0
+
```

Comment:
Are these 3 actually under `rlimit_*`?

Reply:
yes

Reply:
Should they be removed here then?

Reply:
No, `procstat_[prefix_]signals_pending` here is the currently used value. The `procstat_[prefix_]rlimit_signals_pending_hard` (and _soft) down in the `rlimit_*` section are the limits.
Or maybe I'm misunderstanding what you mean.